### PR TITLE
Ft/unsere teams

### PR DESCRIPTION
**Pull request by NelHim**
I have created a Unsere Teams page

### CODE REVIEW

File: src/sections/unsereTeams.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            <div>
+                <h4 class="text-sm font-railwayBold">Clever:</h4>
+                <p class="pt-3 text-sm font-railwayItalic">“d diam nonumy eirmod <br>tempor invidunt ut labore et <br> dolore magna aliquyam erat, <br> sed diam“</p>
+            </div>
+            <div>
+                <h4 class="text-sm font-railwayBold">Izzeddin:</h4>
+                <p class="pt-3 text-sm font-railwayItalic">“d diam nonumy eirmod <br>tempor invidunt ut labore et <br> dolore magna aliquyam erat, <br> sed diam“</p>
+            </div>
+        </div>
+    </div>
```
Comment (ishimwe07) [inline src/sections/unsereTeams.html:81]: Maybe you can consider adding a border for these descriptions.

**Review by ishimwe07 — CHANGES_REQUESTED** [review]
You did great. And it looks good to me.
maybe you can try to add that border and its perfect.

**Review by divinecharlotte — APPROVED** [review]
# Hello @NelHim     😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 

 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏

**Review by NoellaPromise — APPROVED** [review]

**Review by divinecharlotte — APPROVED** [review]

**Review by ishimwe07 — APPROVED** [review]
Approved.

**Pull request merged**